src/components/productForList/ProductForList.js

```diff
@@ -41,33 +44,37 @@ export const ProductForList = ({ item, token }) => {
   }, [item]);
 
   return (
-    <div
-      className="product-container"
-      aria-label={`${
-        item.howSoon === 7
-          ? 'soon'
-          : item.howSoon === 14
-          ? 'kind of soon'
-          : 'not soon'
-      }`}
-    >
-      <input
-        type="checkbox"
-        value="Bought"
-        checked={isBought}
-        disabled={isBought}
-        onChange={handleCheck}
+    <div className="product-container">
+      <div
+        className="color-mark"
+        aria-label={`${
+          item.howSoon === 7
+            ? 'soon'
+            : item.howSoon === 14
+            ? 'kind of soon'
+            : 'not soon'
+        }`}
```

Comment:
¿Creo que tenía una función para eso: Jairo, verdad @JGitHub25 ? Considero, es "más claro". 😄

Reply:
Sí... creo que está en lo del Issue 12 😅

Reply:
Efectivamente! Sin embargo, no es **BLOCKING** para hacer merge.

Como es una función, no afectaría la funcionalidad del issue, solo hacerlo más claro puedes añadirla, por favor 😄 

```javascript
export const setProductStatus = (isInactive, howSoon) => {
  if (isInactive) {
    return 'inactive';
  } else if (howSoon < 7) {
    return 'soon';
  } else if (howSoon >= 7 && howSoon <= 30) {
    return 'kind of soon';
  } else if (howSoon > 30) {
    return 'not soon';
  }
};
```

[imagen extraída](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/blob/8655d44865b85d0f56dc47667d11e4298a3bac50/src/components/ProductForList.js) del PR del issue 12.

![image](https://user-images.githubusercontent.com/8689897/156907365-d5daac44-0672-4727-b573-419b4a75e579.png)

Reply:
cc: @JGitHub25